charts/rhtap-tpa/Chart.yaml

```diff
@@ -3,9 +3,9 @@ apiVersion: v2
 name: rhtap-tpa
 description: RHTAP Trusted Profile Analyzer
 type: application
-version: 0.0.1
+version: 0.1.0
```

Comment:
I think this value should track the TPA version being installed rather than the TPA chart version.

Reply:
I'll merge as is, we can discuss later what makes more sense.